seonshine_frontend/src/pages/menuManagement/index.tsx

```diff
@@ -0,0 +1,200 @@
+                  },
+                },
+              }}
+              InputProps={{
+                startAdornment: (
+                  <InputAdornment position="start">
+                    <SearchIcon />
+                  </InputAdornment>
+                ),
+              }}
+            />
+          </Stack>
+          <Typography
+            variant="h4"
+            component="h3"
+            className="my-8"
+          >
+            Menu List
+          </Typography>
+        </Box>
+        <Stack className="grid grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4">
+          {filteredDishes?.map((dish, index) => (
+            <Stack
+              key={dish.name + index}
+              className="rounded-md p-6 m-2 box-border cursor-pointer bg-white transition-transform transform hover:scale-105 hover:shadow-lg"
+              direction="column"
+              gap={2}
+              onClick={() => handleOpenModal(dish)}
+            >
+              <Box className="w-full h-48 md:h-64 bg-gray-200 flex items-center justify-center overflow-hidden rounded-md">
+                {dish.image_url ? (
+                  <img
+                    src={`${avatarBaseURL}${dish.image_url}`}
+                    className="h-full w-full object-cover"
+                    alt={dish.name}
+                  />
+                ) : (
+                  <Stack className="w-full h-full items-center ">
+                    <RestaurantRounded
+                      className="w-full h-1/2 opacity-30"
+                      fontSize="large"
+                    />
+                  </Stack>
+                )}
+              </Box>
+              <Typography className="font-bold">{dish.name}</Typography>
+            </Stack>
+          ))}
+          <Stack
+            className="rounded-md p-6 m-2 box-border bg-gray-200 cursor-pointer opacity-50 transition-transform transform hover:scale-105 hover:shadow-lg"
+            direction="column"
+            justifyContent="center"
+            alignItems="center"
+            gap={2}
+            onClick={handleAddItem}
+          >
+            <Box
+              className="w-full h-48 md:h-64 flex items-center justify-center overflow-hidden rounded-md"
+              onClick={handleOpenCreateModal}
+            >
+              <AddCircleRoundedIcon
+                className="w-1/2 h-1/2 opacity-30"
+                fontSize="large"
+              />
+            </Box>
+          </Stack>
+        </Stack>
+      </Stack>
+      {isModalOpen && selectedItem && (
+        <ModalMenuItem
+          isOpen={isModalOpen}
+          setIsModalOpen={setIsModalOpen}
+          selectedItem={selectedItem}
+          setSelectedItem={setSelectedItem}
+          isCreateModal={false}
+          item_id={selectedItem.item_id}
+        />
+      )}
+      {isCreateModelOpen && (
+        <ModalMenuItem
+          isOpen={isCreateModelOpen}
+          setIsModalOpen={setIsCreateModelOpen}
```

Comment:
Shouldn't update parent directly in child component. Define onCloseModal with params. Ex: `onCloseModal(isModifySuccess)`.